compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -4340,24 +4338,6 @@ static void unsignedIntCompareNarrower(TR::Node * node, TR::Simplifier * s, TR::
                   dumpOptDetails(s->comp(), "Integer Compare Narrower: found both children c2i in method %s\n", s->comp()->signature());
                   }
                }
-            else if (secondOp == TR::iuconst)
```

Comment:
Why is this removed? shouldn't it be an iconst?

Reply:
This is a mistake. The `TR::iuconst` in this `IntComp` pattern matching should be mapped to `TR::iconst` in this case. I have fixed this problem in cdd352b